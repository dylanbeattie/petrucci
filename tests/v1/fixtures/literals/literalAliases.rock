(all aliases for literals work)
say mysterious
say "aliases for null"
say null
say nothing
say nowhere
say nobody
say gone

say "aliases for empty string"
say empty

say "aliases for true"
say true
say right
say yes
say ok

say "aliases for false"
say false
say wrong
say no
say lies
