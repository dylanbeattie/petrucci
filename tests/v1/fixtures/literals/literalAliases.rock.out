mysterious
aliases for null
null
null
null
null
null
aliases for empty string

aliases for true
true
true
true
true
aliases for false
false
false
false
false
